lib/repl.js

```diff
@@ -646,27 +643,30 @@ function REPLServer(prompt,
 
     if (typeof e === 'object' && e !== null) {
       overrideStackTrace.set(e, (error, stackFrames) => {
-        let frames;
+        let frames = stackFrames;
         if (typeof stackFrames === 'object') {
           // Search from the bottom of the call stack to
           // find the first frame with a null function name
-          const idx = ArrayPrototypeFindIndex(
-            ArrayPrototypeReverse(stackFrames),
-            (frame) => frame.getFunctionName() === null,
+          const idx = stackFrames.findLastIndex(
```

Comment:
```suggestion
          const idx = ArrayPrototypeFindLastIndex(
            stackFrames,
```

Reply:
This method does not exist as primordial right now.

Reply:
cc @aduh95 looks like this one needs to be added then

Reply:
That's because that method can be disabled using a V8 flag:

https://github.com/nodejs/node/blob/71b9f792237b39f283eff4cd7ed35212e8669a45/deps/v8/src/flags/flag-definitions.h#L251